src/mscorlib/src/System/IO/StreamReader.cs

```diff
@@ -737,7 +756,7 @@ public override String ReadLine()
                     i++;
                 } while (i < charLen);
                 i = charLen - charPos;
-                if (sb == null) sb = new StringBuilder(i + 80);
+                if (sb == null) sb = AcquireStringBuilder(i + 80);
```

Comment:
When done using the StringBuilder object in each method, please make sure to clear it so that it doesn't continue to hold the method's data.